### ISSUE TITLE
Actually update to use symfony/process ^5.0

### ISSUE THREAD

**Issue by repat** [mekras/php-speller#22]
```
Call to undefined method Symfony\Component\Process\Process::inheritEnvironmentVariables()
(vendor/mekras/php-speller/src/ExternalSpeller.php)
```

```
Call to undefined method Symfony\Component\Process\Process::setCommandLine()
(vendor/mekras/php-speller/src/ExternalSpeller.php)
```

![image](https://user-images.githubusercontent.com/516807/81054301-a6e26d00-8f1a-11ea-876c-6b26e2072688.png)

-- https://github.com/symfony/process/blob/master/CHANGELOG.md

Can you update, so it doesn't only allow v5 but also works with it? :-)

Thanks!

**Comment by icanhazstring** [comment]
Well, thats unfortunate. I am sorry.
Will check it right away. Thanks for reporting!

**Comment by icanhazstring** [comment]
This will result in a new minor version. But its fine.
So I will drop some supported version since they are no longer supported on their own anyways.

**Comment by icanhazstring** [comment]
@repat I created a PR fixing some stuff, but also updating some implementation and requirements.
If you would be so kind to review if this works for you. Thanks 👍

**Comment by icanhazstring** [comment]
Fixed with #23 and tagged as `2.1.0`
